src/pkg/reg/adapter/harbor/base/adapter.go

```diff
@@ -139,6 +140,15 @@ func (a *Adapter) Info() (*model.RegistryInfo, error) {
 	return info, nil
 }
 
+func buildProjectsQuery(projects map[string]*Project) string {
```

Comment:
if it‘s not a util func, suggest using nested function.

Reply:
moved.